dipy/tests/test_scripts.py

```diff
@@ -121,3 +122,25 @@ def test_qb_commandline():
                '--out_file', 'tracks300.trk']
         out = run_command(cmd)
         assert_equal(out[0], 0)
+
+@nt.dec.skipif(no_mpl)
+def test_qb_commandline_output_path_handling():
+    with InTemporaryDirectory():
+        # Create temporary subdirectory for input and for output
+        os.mkdir('work')
+        os.mkdir('output')
+
+        os.chdir('work')
+        tracks_file = get_data('fornix')
+
+        # Need to specify an output directory with a "../" style path
+        # to trigger old bug.
+        cmd = ["dipy_quickbundles", tracks_file, '--pkl_file', 'mypickle.pkl',
+               '--out_file', '../output/tracks300.trk']
```

Comment:
I am not sure that this will work well when testing on a Windows platform (because the path separator is hard coded here). I am running a test of that over here: https://ci.appveyor.com/project/arokem/dipy/build/1.0.649, but it might take a couple of hours to finish running.